src/python/antares_xpansion/sensitivity_driver.py

```diff
@@ -24,13 +25,16 @@ def launch(
         structure_path,
         json_sensitivity_out_path,
         sensitivity_log_path,
+            xpansion_simulation_output
     ):
         """
         launch sensitivity analysis
         """
         self.simulation_output_path = self._get_simulation_output_path(
             simulation_output_path
         )
+        with zipfile.ZipFile(self.simulation_output_path, 'r') as output_zip:
+            output_zip.extractall(xpansion_simulation_output)
```

Comment:
must extract all ?
what about the final sensitivity outputs, will they be compressed with other xpansion outputs